Directory.Build.targets

```
@@ -22,7 +22,10 @@
 
   <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' ">
     <PackageId Condition=" '$(PackageId)' == '' ">$(AssemblyName)</PackageId>
-    <IsPackable Condition="'$(IsPackable)' == '' AND ( '$(IsTestProject)' == 'true' OR '$(IsTestAssetProject)' == 'true' OR '$(IsBenchmarkProject)' == 'true' OR '$(IsSampleProject)' == 'true' ) ">false</IsPackable>
+    <IsPackable Condition="'$(IsPackable)' == '' AND ( '$(IsTestProject)' == 'true' OR '$(IsTestAssetProject)' == 'true' OR '$(IsBenchmarkProject)' == 'true' OR '$(IsSampleProject)' == 'true' OR '$(IsReferenceAssemblyProject)' == 'true' ) ">false</IsPackable>
+    <HasReferenceAssembly Condition="'$(OutputType)' == 'exe'">false</HasReferenceAssembly>
```

Is it your intention to prevent users from setting this via .csproj? If not, make these conditional
```suggestion
    <HasReferenceAssembly Condition=" '$(HasReferenceAssembly)' == '' AND '$(OutputType)' == 'exe'">false</HasReferenceAssembly>
```

---

Do you think we would ever produce ref for an exe?

---

It's also set to `true` in props for `.csproj` files